tsconfig.json

```diff
@@ -18,6 +18,8 @@
       "components*": ["./src/components*"],
       "hooks": ["./src/hooks"],
       "hooks*": ["./src/hooks*"],
+      "images":["./src/images"],
+      "images*":["./src/images*"],
```

Comment:
Are these indented or no? I'm reviewing this from my phone and it looks indented lol